src/RetryAnnotationTrait.php

```diff
@@ -23,46 +26,39 @@ private function getRetryAttemptsAnnotation()
             $retries = $annotations['class']['retryAttempts'][0];
         }
 
-        return $this->validateRetryAttemptsAnnotation($retries);
+        return $this->parseRetryAttemptsAnnotation($retries);
     }
 
-    /**
-     * @return int
-     * @throws LogicException
```

Comment:
I removed the occurrences of `@throws LogicException` since it's a LogicException... It's not an exception meant to be caught. It's closer to a `CompileException` in Java: it requires an immediate action from the developer to fix the code as opposed to a `RuntimeException`.

I also replaced `LogicException` by `InvalidArgumentException` since I think they are a better fit here